test/specs/core.scale.tests.js

```diff
@@ -73,6 +73,22 @@ describe('Core.scale', function() {
 
 			expect(lastTick(chart).label).toBeUndefined();
 		});
+
+		it('should display multiline-labels correctly', function() {
```

Comment:
nit: "multiline labels" or "multi-line labels" would both be fine. current hyphen location is unexpected